delta/plugins/search/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/search/SearchSpec.scala

```diff
@@ -159,14 +170,28 @@ class SearchSpec
       esClient.bulk(bulkSeq, Refresh.WaitFor).accepted
     }
 
-    "search all indices" in {
-      val results = search.query(jobj"""{"size": 100}""", Query.Empty)(bob).accepted
-      extractSources(results).toSet shouldEqual projections.flatMap(createDocuments).toSet
+    "search all indices accordingly to Bob's full access" in {
+      val results = search.query(matchAll, noParameters)(bob).accepted
+      extractSources(results).toSet shouldEqual allDocuments
+    }
+
+    "search only the project 1 index accordingly to Alice's restricted access" in {
+      val results = search.query(matchAll, noParameters)(alice).accepted
+      extractSources(results).toSet shouldEqual project1Documents
+    }
+
+    "search within an unknown suite" in {
+      search.query(Label.unsafe("xxx"), matchAll, noParameters)(bob).rejectedWith[UnknownSuite]
+    }
+
+    "search within a suite accordingly to Bob's full access" in {
+      val results = search.query(proj2Suite, matchAll, noParameters)(bob).accepted
+      extractSources(results).toSet shouldEqual project2Documents
     }
 
-    "search only indices the user has access to" in {
-      val results = search.query(jobj"""{"size": 100}""", Query.Empty)(alice).accepted
-      extractSources(results).toSet shouldEqual createDocuments(projectionProj1).toSet
+    "search within a suite accordingly to Alice's restricted access" in {
+      val results = search.query(proj2Suite, matchAll, noParameters)(alice).accepted
+      extractSources(results).toSet shouldEqual Set.empty
```

Comment:
I think it's better to have a positive assertion here, otherwise this could pass if a user with restricted access always returns no results